### PR TITLE
Getter for the language sef

### PR DESCRIPTION
**Pull request by saharin88**
I miss for this method

### CODE REVIEW

**Comment by infograf768** [comment]
Why not using directly `LanguageHelper::getLanguages('sef');` ?

**Comment by alikon** [comment]
Can you please add more information like  testing instruction ect

**Comment by saharin88** [comment]
> Can you please add more information like testing instruction ect

$currentLanguage = Factory::getLanguage();
$sef             = $currentLanguage->getSef();

File: libraries/src/Language/Language.php [hunk 1]
```diff
@@ -1031,6 +1031,20 @@ public function getErrorFiles()
 		return $this->errorfiles;
 	}
 
+	/**
+	 * Getter for the language sef
+	 *
+	 * @return string  The language sef.
+	 *
+	 * @since 1.0
```
Comment (infograf768) [inline libraries/src/Language/Language.php]: ` * @since  __DEPLOY_VERSION__`

**Comment by infograf768** [comment]
Looks useful to me. Test OK.
@HLeithner

**Comment by HLeithner** [comment]
Sorry no new features for j3 please rebase to to 4.1. thanks

**Comment by saharin88** [comment]
> 
> 
> Sorry no new features for j3 please rebase to to 4.1. thanks

Why, I do not understand, why wait until the release of joomla 4 (unknown when) ?

**Comment by HLeithner** [comment]
Joomla 3 is in feature freeze for a long time. All focus should go into the release of joomla 4.0.

**Comment by saharin88** [comment]
> 
> 
> Joomla 3 is in feature freeze for a long time. All focus should go into the release of joomla 4.0.

I have no idea when the 4th version of Joomla will be released. Instead, there is a working stable 3rd version, and I can not understand why restrict its improvement ?

**Comment by rdeutz** [comment]
> I can not understand why restrict its improvement ?

We have decided not to accept new features for the 3.x series. The reason are the limited resouces we have.

**Comment by rdeutz** [comment]
I tried to rebase your PR but it seems to me that it is better you make a new PR for the change against the 4.1-dev branch. I am Closing this PR.

**Comment by saharin88** [comment]
> 
> 
> I tried to rebase your PR but it seems to me that it is better you make a new PR for the change against the 4.1-dev branch. I am Closing this PR.

after your decisions there is no desire to do anything

**Pull request closed without merging**